src/Blueprint/Blueprint/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.2.1
```

Comment:
@filizt you can get rid of this header and put the below line below `## Upcoming Release`